cmd/tailscale/cli/web.html

```diff
@@ -158,7 +172,11 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
 		}
 		const url = res["url"];
 		if (url) {
-			document.location.href = url;
+			if(isUnraid) {
```

Comment:
Space after the if please (matching the existing convention in the file).

Reply:
Changed 👍